### DISCUSSION TITLE
PTCG 熱潮

### DISCUSSION THREAD

**Discussion by CeBru13** [1101-GS4719-C/Python-Community-GS4719-C#2681] [General]
相信大家都知道最近很紅的寶可夢卡牌手機遊戲，我個人也有玩，目前的看法是，玩法有點單一，打幾局就不想打了，每天就等抽卡包，是現在沒什麼創意嗎，還是只是公司想靠抽卡包賺錢?

**Comment by kzx-creat** [comment]
TCGP 我只看過別人玩，我覺得有點像百貨公司裏面可以放卡片的機台遊戲

**Comment by Taoyi-Chiang** [comment]
PTCG的設計有比較簡化，我覺得這種設計滿適合睡前或短暫休息的時候打個一兩局，抽抽卡的。
至於玩法單一與否，我覺得和其他玩家的對戰過程，會看到很多不同的組牌思路，其實也滿有趣。

**Comment by DarsooK** [comment]
我認為才上架沒天就這樣批評大可不必，等環境更新到實卡環境吧，雖然沒玩PTCG，但隔壁MD剛推出時也比實卡慢個幾年，現在也快追回來了。
